pkg/menu/menu.go

```diff
@@ -119,27 +127,18 @@ func NewInputWindow(introwords string, isValid validCheck, uiEvents <-chan ui.Ev
 
 // NewCustomInputWindow creates a new ui window and displays an input box.
 func NewCustomInputWindow(introwords string, wid int, ht int, isValid validCheck, uiEvents <-chan ui.Event) (string, error) {
```

Comment:
I'm wondering if the New and the Display should not be two things. 
New would give the window and the user would then call input processing. 
THink about how exec.Command and exec.Run or exec.Start do this.

Reply:
I think I like the way April set it up, just because it feels like how other languages collect user input. It feels like doing something like `val = input("Enter your value: ")`, and then just waiting until the user inputs something before continuing.

Except I also like how the user can customize the exec options between `exec.Command()` and `exec.Run()`. Maybe the user wants to set a custom validation function on the input box.

Maybe the compromise is to have some widgets that are entirely customizable by the caller, and others that are just like a `val = input()` function call. CustomInputWindow would be like the latter, but something like the Progress widget in [the new PR](https://github.com/u-root/webboot/pull/136) gives the caller more control.